test/hardhat/integration/bitcoin-genesis.test.js

```
@@ -0,0 +1,284 @@
+
+    const ProxyAdmin = await ethers.getContractFactory("ProxyAdmin");
+    const proxyAdmin = await ProxyAdmin.deploy();
+    await proxyAdmin.waitForDeployment();
+    console.log(`ProxyAdmin deployed at: ${proxyAdmin.target}`);
+
+    const currentTimestamp = await time.latest();
+    console.log(`Current timestamp: ${currentTimestamp}`);
+
+    const initializeArgs = await generateBootstrapInitData();
+
+    const Proxy = await ethers.getContractFactory("TransparentUpgradeableProxy");
```

Biome (1.9.4)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)